src/main/java/com/ft/methodearticlemapper/resources/PostContentToTransformResource.java

```diff
@@ -40,15 +40,26 @@ public PostContentToTransformResource(EomFileProcessor eomFileProcessor) {
 
 	@POST
 	@Timed
-	@Path("/{uuidString}")
+	@Path("/content-transform/{uuidString}")
```

Comment:
Will this still be kept after PAM will call the /map endpoint? Can we make content-preview to use this endpoint?